### PR TITLE
Update lib for protocol version 3.20

### PR DESCRIPTION
**Pull request by kenzieschmoll**
@devoncarew

### CODE REVIEW

**Comment by devoncarew** [comment]
Looks like some CI failures:

https://travis-ci.org/dart-lang/vm_service_drivers/jobs/548777420#L350

File: dart/tool/service_undocumented.md [hunk 1]
```diff
@@ -17,29 +17,6 @@ Success _requestHeapSnapshot(string isolateId, string roots, bool collectGarbage
...
-
-```
-@Object _getInstances(
-  string isolateId,
-  string classId,
-  int limit
-)
-```
-
 Returns a ServiceObject (a specialization of an ObjRef).
```
Comment (devoncarew) [inline dart/tool/service_undocumented.md]: This line can be deleted (it goes with the proceeding declaration).

**Comment by devoncarew** [comment]
The java CI failure (https://travis-ci.org/dart-lang/vm_service_drivers/jobs/548777421#L380) is a known issue - not related to this PR.

File: dart/tool/service_undocumented.md (hunk 1 above)
Comment (kenzieschmoll) [reply dart/tool/service_undocumented.md]: deleted.

**Pull request merged**